packages/data-point/lib/entity-types/entity-collection/resolve.test.js

```diff
@@ -54,38 +54,23 @@ describe('Collection entity type checking', () => {
   })
 })
 
-describe('entity.collection.value', () => {
-  test('should resolve value Transform', () => {
-    return transform('collection:a.1', testData).then(acc => {
-      expect(acc.value).toEqual([
-        {
-          d1: 2
-        },
-        {
-          d1: 4
-        }
-      ])
-    })
-  })
-})
-
 describe('entity.collection.map', () => {
   test('should resolve map Transform', () => {
-    return transform('collection:b.1', testData).then(acc => {
+    return transform('collection:b.1', testData.a.d).then(acc => {
```

Comment:
changes on these tests change because the entity itself does not resolve the value anymore, since value gets delegated to the base entity, on the tests we have to pass the resolved value

Reply:
one option was to change all the entities to remove the value reducer but then the entity definitions would lose a bit of context as examples on their own. Eventually, I want move away from using them for tests and only use instance entities